stores/sql/mysql/main.go

```diff
@@ -539,6 +473,62 @@ func (tx *MainDatabaseTx) InvalidateSlabHealthByFCID(ctx context.Context, fcids
 	return res.RowsAffected()
 }
 
+func (tx *MainDatabaseTx) InsertDirectoriesDeprecated(ctx context.Context, bucket, path string) (int64, error) {
```

Comment:
I think considering our plans for v3 and the fact that the directories table is completely gone now it's probably fine to clear the migration code and remove the following leftovers

`object/path.go`
`object/path_test.go`
`InsertDirectories` (4 times)
`InsertDirectoriesDeprecated`

Reply:
This latest version has been play tested and migration tested. We've always had the rule not to touch previous migrations which is one of the reason we have `InsertDirectoriesDeprecated` in the first place. Getting rid of the migration that introduces `db_bucket_id` might potentially even cause this new migration to fail because it expects that field to be there. Wouldn't you agree? I would F/U with that because it is indeed a decent amount of files and lines that can be removed but we have to reset migrations anyway, might as well do it in one go.